src/components/AdminNicety.js

```diff
@@ -23,19 +22,16 @@ class AdminNicety extends React.Component {
             target_id: this.props.target_id,
             faculty_reviewed: this.state.reviewedValue,
         }
-        $.ajax({
-            url: this.props.admin_edit_api,
-            data: data,
-            dataType: 'json',
-            type: 'POST',
-            cache: false,
-            success: function () {
-                this.setState({noSave: true});
-            }.bind(this),
-            error: function (xhr, status, err) {
-                console.log(err)
-            },
-        });
+        fetch(this.props.admin_edit_api, {
```

Comment:
Something to double-check, although I think this is already doing the right thing: does this `fetch` include all the necessary cookies? I think we're on the same origin, and the default behavior should be to send cookies in that case, but let's be certain.